CODEOWNERS

```diff
@@ -125,5 +125,11 @@ go.mod @fleetdm/go
 ##############################################################################################
 /.github/workflows/ @lukeheath @georgekarrv @sharon-fdm @mostlikelee
 
+##############################################################################################
+# 🌐 Fleet-maintained apps
+##############################################################################################
+/ee/maintained-apps/ @eashaw
+/server/mdm/maintainedapps/ @eashaw
+
```

Comment:
```suggestion
```

This code is being retired as a source of truth for the app library, but has Go code in it that we'll be pulling elsewhere and don't want to hold up that process.